pkg/epp/controller/inferencemodel_reconciler.go

```diff
@@ -41,25 +43,20 @@ type InferenceModelReconciler struct {
 }
 
 func (c *InferenceModelReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	if req.Namespace != c.PoolNamespacedName.Namespace {
-		return ctrl.Result{}, nil
-	}
-
-	logger := log.FromContext(ctx)
-	loggerDefault := logger.V(logutil.DEFAULT)
-	loggerDefault.Info("Reconciling InferenceModel", "name", req.NamespacedName)
+	logger := log.FromContext(ctx).WithValues("Request.Namespace", req.Namespace, "Request.Name", req.Name).V(logutil.DEFAULT)
```

Comment:
Thanks Nir, I am updating the logging in https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/393, do you mind reverting the changes in the Reconcile func just so we don't cause lots of merge conflicts?